SIGs/GRC/publications/Confidential Payload Governance.md

```diff
@@ -0,0 +1,111 @@
+    
+  * Automated tools MUST be used to continuously monitor and manage third-party software libraries and dependencies for known vulnerabilities. Dependencies MUST be kept updated and strict vetting processes applied, including code signature and build metadata checks if available, when ingesting dependencies. Dependency management MUST continue for the duration of the application’s deployment.  
+  * Include Bills of Materials as part of generated software; the BOMs SHOULD **\[g\]** be verified by the Verifier post-deployment.
+
+* **Secure Integration, Configuration and Deployment**  
+    
+  * **Minimizing the Payload TCB:** Care MUST be taken to include only the minimum amount of code in each TEE that is necessary for the functionality encapsulated by that TEE.  
+  * **Trusted Root Store Hygiene:** The contents of the Root Stores, if any, used by the Confidential Payload, MUST be carefully curated and safeguarded against tampering. If used as a configuration parameter (as opposed to being hard-coded into the Payload), the Trusted Root Store measurement MUST be included in the remote attestation process.  
+  * **Cryptography Hygiene:** The choice of cryptographic ciphers, key lengths and modes MUST be carefully curated and safeguarded against tampering. If configurable (as opposed to being hard-coded into the Payload), these choices MUST be included in the remote attestation process.  
+  * **Cryptographic Key Hygiene:** All cryptographic keys procured from external Key Vaults MUST only be released to properly authorized requesting TEEs, usually based on the results of successful Remote Attestation and delivered to the requesting TEEs using a secure transport.  
+  * **Security-Sensitive Configuration Hygiene:** All security-sensitive configuration MUST be included in the remote attestation process.  
+  * **Verifier Hygiene:** The Verifier policies MUST match the most recent versions of deployed Payloads; older vulnerable Payloads MUST be phased out in a timely fashion following successful deployment of up-to-date versions.
+
+# Governance Expectations Summary
+
+The numbers in the left column refer to **\[1\]**. Rows listed as N/A indicate that corresponding expectations are listed under different Patterns documents.
+
+| \# | Description |
+| :---- | :---- |
+| 2-6, 8, 10-11, 13 | N/A |
+| 1 | Supply accurate and up-to-date component-specific guidance for secure software development. |
+| 7 | Update Verifier policies with the Reference Values of newly deployed packages. |
+| 9, 14 | Evidence of establishment of and adherence to secure design & development practices Evidence of securely managed build environment Evidence of robust supply chain and dependency management Evidence of secure integration, configuration and deployment practices |
+| 12 | Provide evidence of measurements of newly deployed packages being recognized as valid. |
+| 15 | Evidence of requiring and validating that expectations set out in (9, 14\) are satisfied. |
```

Comment:
Does this section and table still belong in the document? I think this once referred to a diagram? I also don't see the diagram in the original google doc.

Reply:
I see the diagram is in this doc: https://docs.google.com/document/d/1X10ymjFgUy5NGPKs6Z3F_ERPwrgRuK23-9F2h3-LbWw/edit?tab=t.0#heading=h.wir5oj6loeqt 
That document should be part of this PR. 
I think the diagram should be embedded in each of the 3 patterns documents that are part of this PR.

Reply:
Should we review that doc first in a separate PR and merge it first?